spec/peeps_spec.rb

```diff
@@ -0,0 +1,31 @@
+require 'peeps'
+
+describe Peeps do
+
+  describe '.all' do
+    it 'returns a message from the db' do
+      connection = PG.connect(dbname: 'chitter_test')
+      connection.exec("INSERT INTO peeps (message) values ('This is a peep!');")
+      result = connection.exec('SELECT * FROM peeps;')
+      expect(result[0]['message']).to eq "This is a peep!"
+    end
+  end
+
+  describe '.add' do  
+    it 'adds a new peep to the db' do
+      new_peep = "Hi everyone"
+      connection = PG.connect(dbname: 'chitter_test')
+      Peeps.add(new_peep)
+      result = connection.exec("SELECT * FROM peeps WHERE message = 'Hi everyone';")
+      expect(result[0]['message']).to eq "Hi everyone"
+    end
+  end
+
+  describe '.add' do
+    it 'adds a peep with a date' do
+      # need to stub in a date for the test
```

Comment:
There's a gem called TimeCop that might help you here, though I'd have thought your test would be fine (most of the time) since the datestamp is accurate to minutes, rather than seconds.  I.e. You could probably do something like this...

```ruby
describe '.add' do  
    it 'adds a new peep to the db' do
      new_peep = "Hi everyone"
      connection = PG.connect(dbname: 'chitter_test')
      datestamp = Time.new
      Peeps.add(new_peep)
      result = connection.exec("SELECT * FROM peeps WHERE message = 'Hi everyone';")
      expect(result[0]['date']).to eq "
          #{datestamp.hour}:#{datestamp.min}, #{datestamp.day}.#{datestamp.month}.#{date.year}
       "
    end
  end
```

Drop me a line on Slack if you want to chat about this!